data/org.gnome.shell.gschema.xml.in

```diff
@@ -18,6 +18,15 @@
         using the Alt-F2 dialog.
       </description>
     </key>
+    <key name="enable-app-center" type="b">
```

Comment:
I guess it's ok to ignore the former name enable-app-center if this is for development / testing purposes, even though that will break current installations using a non default here, so I guess it's ok as long as we don't set this key from anywhere else (e.g. image builder), which according to github search it's **not** the case, so ok.